docs/feature_leader_key.md

```diff
@@ -5,7 +5,8 @@ If you've ever used Vim, you know what a Leader key is. If not, you're about to
 That's what `KC_LEAD` does. Here's an example:
 
 1. Pick a key on your keyboard you want to use as the Leader key. Assign it the keycode `KC_LEAD`. This key would be dedicated just for this -- it's a single action key, can't be used for anything else.
-2. Include the line `#define LEADER_TIMEOUT 300` in your config.h. The 300 there is 300ms -- that's how long you have for the sequence of keys following the leader. You can tweak this value for comfort, of course.
+2. Include the line `#define LEADER_TIMEOUT 300` in your `config.h`. The 300 there is 300ms -- that's how long you have for the sequence of keys following the leader. You can tweak this value for comfort, of course.
+   * By default, thi timeout is how lonk after hitting `KC_LEAD` you to complete your entire sequence. This may be very low for some people. So you may want to increase this timeout.  Optionally, you may want to switch to the `LEADER_PER_KEY_TIMING` option, which resets the timeout after each key is tapped. 
```

Comment:
Here are some typos: 
`* By default, thi timeout is how lonk after hitting `
thi should be the and lonk should be long. I also think that there is no need to say by default. I know that you mean without leader per key applied but I think there is no need to be so specific.

Regarding the last sentence, as in the  other file (config_options.md) I think it makes no sense to say `switch to` because you are not actually switching, but adding an extra configuration. Also so many may start to look redundant to me. What do you think about this rewording: 

Optionally you can add the `LEADER_PER_KEY_TIMING` option, which resets the timeout after each key is tapped (more details below).